File: CPP/High_Voltage/cPlayer.cpp
#include"GlobalVariables.h"

bool cPlayer::Alive() {
    return cPlayer::m_bAlive;
}

void cPlayer::Alive(bool bNowAlive) {
    cPlayer::m_bAlive = bNowAlive;
}

int cPlayer::Row() {
    return cPlayer::m_iRow;
}

void cPlayer::Row(int iNowRow) {
    cPlayer::m_iRow = iNowRow;   
}

int cPlayer::Column() {    
    return cPlayer::m_iColumn;
}

void cPlayer::Column(int iNowCol) {
    cPlayer::m_iColumn = iNowCol;   
}

File: HTML/High_Voltage_JS/cPlayer.js
class cPlayer {
   //Private:
   #m_bAlive = true;
   #m_iRow;
   #m_iColumn;

   //Public:
   Alive() {
      return cPlayer.#m_bAlive;
   }

   Alive(bNowAlive) {
      cPlayer.#m_bAlive = bNowAlive;
   }

   Row() {
      return cPlayer.#m_iRow;
   }

   Row(iNowRow) {
      cPlayer.#m_iRow = iNowRow;
   }

   Column() {
      return cPlayer.#m_iColumn;
   }

   Column(iNowCol) {
      cPlayer.#m_iColumn = iNowCol;
   }
}
